pkg/cmd/cli/cmd/process.go

```diff
@@ -37,17 +37,17 @@ func injectUserVars(cmd *cobra.Command, t *api.Template) {
 func NewCmdProcess(f *Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "process -f filename",
-		Short: "Process template into config",
-		Long: `Process template into a config specified in filename or stdin
+		Short: "Process template into list of resources",
+		Long: `Process template into a lis of resources specified in filename or stdin
 
 JSON and YAML formats are accepted.
 
 Examples:
```

Comment:
@kargakis this is other PR that is currently being merged, will have to be fixed afterwards or you can add the fix into your golint PR.